_data/topnav.yml

```diff
@@ -15,7 +15,9 @@ topnav:
       url: /blog/
     - title: Help
       url: /help/
+    - title: Related Tooling
```

Comment:
Just a note that we are now adding even more material to an already crowded bar, perhaps we should move this issue (https://github.com/pulp/pulpproject.org/issues/182) to consolidate some/all into a dropdown menu instead should move up in priority so as not to have a bad user experience on smaller screens/mobile devices?

Reply:
Yeah, I'm wondering about this too.

As much as I hate delaying a merge, it seems like Brian did the last big restructuring of the site a0fcf28d4654afed635c05ff42ef44eed8e5fa8e https://pulp.plan.io/issues/1862 . So let's wait for his feedback.

Reply:
I'd be ok w/ not delaying merge as this has an issue to track the work separately already, it's just that it is on github and not plan.io and has a tendency to get forgotten, so we would want to make sure it stays on the radar.

Reply:
@CodeHeeler I agree. Please re-review in the meantime, I'm waiting for juan-cabrera to reply also.